lightstep/envoy_propagator.py

```diff
@@ -1,15 +1,14 @@
 from __future__ import absolute_import
 
-import struct
+from base64 import b64encode
```

Comment:
Maybe use standard_b64encode and standard_b64decode instead of the default b64encode/b64decode.